pybamm/models/submodels/thermal/x_full.py

```diff
@@ -48,13 +48,27 @@ def set_rhs(self, variables):
         T = variables["Cell temperature"]
         Q = variables["Total heating"]
 
+        # Define volumetric heat capacity
+        rho_k = pybamm.Concatenation(
```

Comment:
Not sure if `rho_k` should be assembled here or in the parameters scripts...

Reply:
Here is fine, but could you do this instead of calling `standard_variables`?
```python3
T_n, T_s, T_p = T.orphans
```